webrtc.html

```diff
@@ -4222,7 +4222,17 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                 </li>
                 <li>
                   <p><a>Create an RTCRtpReceiver</a> with <var>kind</var> and
-                  let <var>receiver</var> be the result.</p>
+                  let <var>receiver</var> be the result. Calls to
+                  <code>createOffer</code> cannot be configured to receive
```

Comment:
Change this from 

    Calls to createOffer cannot be configured to receive multiple RTP encodings. ...

to 

   This specification does not define how to configure an offer to receive multiple RTP encodings ...